dev/WindowsAppSDKTelemetryResource/WindowsAppSDKTelemetryResourceLib/WindowsAppSDKTelemetry.cpp

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <wil/resource.h>
+#include <string>
+
+#define _WINDOWSAPPSDK_PACKAGE_VER_RESOURCEID             10000
+#define _WINDOWSAPPSDK_EXPERIMENTATION_FLAG_RESOURCEID    10001
+
+#define TELEMETRYRESOURCEDLL "WindowsAppSDKTelemetryResource.dll"
+
+inline std::string load_resource_string(const char* dllName, uint32_t id)
+{
+    wil::unique_hmodule module(::LoadLibraryA(dllName));
+
+    if (module)
+    {
+        char buffer[256];
+        if (0 < LoadStringA(module.get(), id, buffer, ARRAYSIZE(buffer)))
+        {
+            return buffer;
+        }
+        else
+        {
+            LOG_LAST_ERROR_MSG("Unable to load resource string, id: %d, error: %d", id, GetLastError());
```

Comment:
Should we log telemetry using non-reunion telemetry so that we can track # of load DLL failures? (or) are there other ways to know that telemetry log failed? Maybe check for null package version from all events with COMMON_WINDOWSAPPSDK_PARAMS?

Reply:
Is there a generic WindowsAppSDK provider that we can log against?

Reply:
Please create one.